packages/frontend-2/components/viewer/explorer/Filters.vue

```diff
@@ -246,4 +230,25 @@ const refreshColorsIfSetOrActiveFilterIsNumeric = () => {
   // removePropertyFilter()
   applyPropertyFilter()
 }
+
+const isRevitProperty = (key: string): boolean => {
```

Comment:
1. This same helper can be used on like 145
2. Dunno how many times this gets invoked (are there thousands of such checks?), but if there are many it would be more performant to have a regexp defined outside of the function and testing the string against that instead

Reply:
This helper is not fully DRY to maintain the original logic structure, but it is better than it was